### PR TITLE
Easy customize

### PR DESCRIPTION
**Pull request by Nepomuceno**
### Prerequisites

- [x] This pull request adds a feature.

### Description
Adding classes to the listing so we can avoid this messing up customization that people might end up doing in `li.html` 

### Issues Resolved

It resolves the fact of anyone trying to configure a custom post list.

#### General

I am including the classes on lists to facilitate the customization of the template.

#### Resources

- [x] If you have changed any SCSS code, run `make release` to regenerate all CSS files

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

File: assets/scss/_content.scss [hunk 1]
```diff
@@ -71,7 +71,7 @@
       @media only screen and (max-width : 768px) {
         margin: 1.6rem 0 1.6rem 0;
       }
-      span {
+      span.date {
```
Comment (luizdepra) [inline assets/scss/_content.scss]: We can remove the `span` here.

File: assets/scss/_content.scss [hunk 2]
```diff
@@ -81,7 +81,7 @@
           text-align: left;
         }
       }
-      a {
+      a.title {
```
Comment (luizdepra) [inline assets/scss/_content.scss]: Remove the `a` here too.

File: assets/scss/_content_inverted.scss [hunk 3]
```diff
@@ -5,7 +5,7 @@ body.inverted {
     .list {
       ul {
         li {
-          a {
+          a.title {
```
Comment (luizdepra) [inline assets/scss/_content_inverted.scss]: Same here.

File: assets/scss/_content_rtl.scss [hunk 4]
```diff
@@ -20,7 +20,7 @@ body.rtl {
   .list {
     ul {
       li {
-        span {
+        span.date {
```
Comment (luizdepra) [inline assets/scss/_content_rtl.scss]: Here too.

**Comment by luizdepra** [comment]
Nice. I just thing we should remove some tags from the SCSS code, so we can use the class with other tags. I've marked the spots that need to change.

**Comment by Nepomuceno** [comment]
Corrected the recommendations thank you!

**Comment by luizdepra** [comment]
Thank you. :)

**Pull request merged**